database-plugins/src/test/java/co/cask/hydrator/plugin/DatabasePluginTestBase.java

```diff
@@ -53,32 +60,46 @@
 /**
  * Database Plugin Tests setup.
  */
-public class DatabasePluginTestBase extends ETLBatchTestBase {
-  public static boolean tearDown = true;
-
-  protected static final long CURRENT_TS = System.currentTimeMillis();
+public class DatabasePluginTestBase extends TestBase {
```

Comment:
to remove core-plugins test jar.  It also fails on my laptop, not sure why it doesn't fail on travis.